docs/metrics-specification.md

```diff
@@ -0,0 +1,302 @@
+# SniTun Server-Side Metrics Specification
```

Comment:
This looks like it is designed to be standalone, and as it does not have any specific logic, I think it should.
All of this looks like a standalone library that Snitun should (optionally) import and use. It could benefit the community better as a standalone solution than being hidden away inside here.

That also gives the option to not have this on the client, as the installation of the lib would be on the server side only.